Employee_details_using _structures/EmployeeDetails.c

```diff
@@ -93,85 +99,75 @@ struct employee* add_employee(struct employee *employee_ptr,int *employee_count)
 }
 
 
-void display_employee_details(struct employee *employee_ptr , int employee_count)
+void display_employee_details(struct employee *employee_ptr)
 {
  	int index;
-	printf("number of employee =%d\n",employee_count);
-	for(index=0;index<employee_count;index++)
-	{
-		printf("Employee    :%d\n",index+1);
-	 	printf("NAME        :%s\n", employee_ptr[index].employee_name);
-	 	printf("ID          :%d\n", employee_ptr[index].employee_id);
-	 	printf("AGE         :%d\n", employee_ptr[index].employee_age);
-	 	printf("DESIGNATION :%s\n\n", employee_ptr[index].employee_designation);
-
-	}
-
+	printf("size of employee =%d\n",sizeof(struct employee));
+	printf("size of employee ptr =%d\n",sizeof(employee_ptr));
+	// for(index=0;index<employee_count;index++)
+	// {
+	//  	printf("NAME        :%s\n", employee_ptr[index].employee_name);
+	//  	printf("ID          :%d\n", employee_ptr[index].employee_id);
+	//  	printf("AGE         :%d\n", employee_ptr[index].employee_age);
+	//  	printf("DESIGNATION :%s\n\n", employee_ptr[index].employee_designation);
+  // }

```

Comment:
already told to remove debug prints and why these lines are commented, then what will you display?

Reply:
I have wrongly committed and changed it in recent commit.